pullrequest.rst

```diff
@@ -119,10 +119,30 @@ homepage.
 Submitting
 ----------
 
-Once you are satisfied with your work you will want to commit your
-changes to your branch. In general you can run ``git commit -a`` and
-that will commit everything. You can always run ``git status`` to see
-what changes are outstanding.
+Once you are satisfied with your work, you will need to commit your
+changes to your branch. Assume you want to add a file which you have changed,
+you will need to enter this command::
+
+   git add <file>
+
+After adding all files of changed, you can commit your changes::
+
+   git commit
+
+Inside the commit editor, commit message's title should start with
+``bpo-NNNNN`` and the major change of this commit, for example::
+
+   bpo-29624: Adds purge step and layout test after uploading files.
+
+Two commands, ``git log`` and ``git status``, provide helpful information
+and detail about commits cand changes.
+
+.. note::
+
+   You will saw hashtag with number in git log, e.g.
+   「bpo-29624: Adds purge step and layout test after uploading files.
+   **(#258)**」, you won't need to add this into your commit title, it will
+   be auto added when making a pull request on GitHub.
```

Comment:
I find this note confusing.  IIUC you are trying to say that the PR number shouldn't be included in the commit message, but why would the PR number show up in `git log` if I didn't include it in the commit message?

Also:
* I wouldn't call the PR num "hashtag"
* will saw -> will see
* Use regular ASCII quote or rst markup for the message